sdk/tables/azure-data-tables/setup.py

```diff
@@ -63,7 +63,7 @@
     ]),
     python_requires=">=3.7",
```

Comment:
unrelated to this PR, but the setup should be updated to support 3.6 and 3.10

Reply:
Got spot - we should fix that and the classifiers while we're here.

Reply:
The 3.6 support is in a separate PR - so I'll merge that after this one.
https://github.com/Azure/azure-sdk-for-python/pull/22564